src/RestaurantButton.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const RestaurantButton = props => {
+  return (
+    <button onClick={props.order} className="btn btn-primary m-2">
```

Comment:
Good to see you passing functions as props as this is something you weren't super confident when we went over it in class, good progress!